sympy/solvers/solveset.py

```diff
@@ -864,6 +868,15 @@ def solveset(f, symbol=None, domain=S.Complexes):
     variable being solved for and instead, uses the `domain` parameter to
     decide which solver to use.
 
+    Notes
+    =====
+
+    N.B. Python interprets 0 and 1 as False and True, respectively, but
```

Comment:
Perhaps. It's not a generic note, it really needs to be emphasized. But I will leave the Note heading to do the emphasis.